### DISCUSSION TITLE
diary/2021/12/26/2021-retrospective/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#135] [Comments]
#  2021년 회고 및 요즘 생각들 

https://zzsza.github.io/diary/2021/12/26/2021-retrospective/

**Comment by zzsza** [comment]
<p>너무 좋은 글입니다. 많은 깨달음을 얻게 됐습니다. 앞으로 성윤님의 커리어, 블로그, 유튜브 모두 응원합니다!</p>

>_This comment was made by Disqus. 2021-12-29 11:18:16 이태호_

**Comment by std-freejia** [comment]
저에 대해서 반성하게 되네요. 좋은 글 공유해 주셔서 감사합니다. 응원합니다!
